__tests__/tailwind.test.js

```diff
@@ -53,213 +55,213 @@ import boxShadow from '../corePlugins/boxShadow';
 import textShadow from '../corePlugins/textShadow';
 
 test('alignContent', () => {
-    expect(alignContent).toEqual(require('./fixtures/outputs/tailwind/alignContent'));
+    expect(alignContent({theme, colors})).toEqual(require('./fixtures/outputs/tailwind/alignContent'));
```

Comment:
`alignContent` does not need `colors` or am I missing something 🤔

Reply:
Actually to save time I passed `theme` and `colors` in all closures, because there were lot of closures. I used the fact that javascript will discard any arguments passed if it is not in use and will not give error or warning.

Reply:
I understand that it easier to just pass them all but I think it is way cleaner to think about what it needs so I would love this to be only the necessary parameters. 
I will do this if you don't find the time.

greetings
Thomas